smallrye-reactive-messaging-kafka/src/main/java/io/smallrye/reactive/messaging/kafka/impl/KafkaSink.java

```diff
@@ -79,6 +80,8 @@ public KafkaSink(Vertx vertx, KafkaConnectorOutgoingConfiguration config, KafkaC
 
         partition = config.getPartition();
         retries = config.getRetries();
+        Long defaultDeliveryTimeoutMs = Long.valueOf(ProducerConfig.configDef().defaultValues().get(ProducerConfig.DELIVERY_TIMEOUT_MS_CONFIG).toString());
```

Comment:
Why do we have to use toString?

Reply:
resolved, `defaultDeliveryTimeoutMs` and `deliveryTimeoutMs` are int now